iree/hal/cuda/graph_command_buffer.c

```diff
@@ -348,8 +348,6 @@ static iree_status_t iree_hal_cuda_graph_command_buffer_update_buffer(
 
   CUdeviceptr target_device_buffer = iree_hal_cuda_buffer_device_pointer(
       iree_hal_buffer_allocated_buffer(target_buffer));
-  CUdeviceptr dst = target_device_buffer +
```

Comment:
iree_hal_buffer_byte_offset(target_buffer) should be added to dstXInBytes

Reply:
like this?

```
      .dstDevice = target_device_buffer,
      .dstXInBytes = target_offset + iree_hal_buffer_byte_offset(target_buffer),
```

Reply:
(I got confused here because some places modify the size argument in-place - would be good to do a scrub of these files and fix that so we aren't mutating args)